mlibrary/catalog-browse
# 2022-09-06 Updates

# Overview
This pull request relates to LIT's update-a-thon!

## NPM
`npm`, `cssnano`, and `postcss` have been updated to their latest versions (as of September 6, 2022).

## `images.css`
`./css/styles/images.css` has been deleted and removed from `./css/browse.css` because the stylesheet related only to the `m-icon` component, which is no longer used in the app.

## Accessibility
ARC Toolkit gave the warning `Accessible text does not contain visible text`. This is because the `aria-label` value did not match the text that was inside the anchor. The text was move all into the anchor, with a `visually-hidden` wrap.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Install the updated packages.
- Run the site and make a css change.
  - Build the CSS.
  - Make sure everything works.
